typing_extensions/src_py3/typing_extensions.py

```diff
@@ -1780,7 +1128,10 @@ class Point2D(TypedDict):
     # Not exported and not a public API, but needed for get_origin() and get_args()
     # to work.
     _AnnotatedAlias = typing._AnnotatedAlias
+# 3.7-3.8
 elif PEP_560:
+    import operator
```

Comment:
Nit: While this import is only used for this branch, I suggest to import at the top for better overview.

Reply:
Done